lib/index.js

```diff
@@ -314,12 +314,22 @@ class GalleryImageArticle extends BaseAsset {
         }
     }
 
+    set_content_dates(value) {
+        this._content_dates = value.map((v) => {
+            if (!(v instanceof Date)) {
+                return new Date(v);
+            }
+
+            return v;
+        });
+    }
+
     to_metadata() {
         const metadata = super.to_metadata();
         Object.assign(metadata, {
             'document': this._document.html(),
             'authors': [this._author],
-            'published': this._date_published.toISOString(),
```

Comment:
I'm guessing that if we *have* to set a date, we can set it to the current date, although you lose deterministic sorting.